docs/package_firewall/faq.md

```diff
@@ -0,0 +1,56 @@
+# Frequently Asked Questions
+
+## What do I do when a dependency fails analysis?
+
+If a package's installation gets blocked by the firewall, possible malicious
+code execution will have been blocked, so there's no immediate need for action.
+
+Should you determine the policy violation to be a false-positive, you can add an
```

Comment:
"false-positive" sounds like we did something wrong, perhaps "acceptable or not relevant"?

Reply:
That's a good point, it might be a "true" positive but people still want to ignore it.

Reply:
I decided to go with "insignificant", since I believe it's still accurate but more concise than "acceptable or not relevant".